app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from os import makedirs, remove, path
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
you can unpack command into the variables

Reply:
what do you mean? Unpack command "mv" to another variables?

Reply:
> you can unpack command into the variables
what do you mean? Unpack command "mv" to another variables?

Reply:
command, file_name, ... = command.split()